sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs

```diff
@@ -78,6 +79,7 @@ version switch
                 ServiceVersion.V2020_06_12 => "2018-11-09",
                 ServiceVersion.V2020_08_04 => "2018-11-09",
                 ServiceVersion.V2020_10_02 => "2018-11-09",
+                ServiceVersion.V2020_12_06 => "2018-11-09",
```

Comment:
I wonder if we should start moving this forward like java does here https://github.com/Azure/azure-sdk-for-java/blob/040045869c2052da4c648dfafc65a7e97588cffe/sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueServiceVersion.java#L19 .

If not for features then maybe bugfixes and improvements that could be gated by service version.

Reply:
We could consider changing this.  I don't think the Queues service has gotten a new feature since 2018, but it's possible there were bugfixes in later versions.

Reply:
If we do so, we'd rather leave service versions that shipped intact and just bump as we add new.

Reply:
So leave all existing service versions the way they are, only bump going forward?

Reply:
yes, that's safe strategy.